pkg/network/node/ovscontroller.go

```diff
@@ -261,9 +262,16 @@ func (oc *ovsController) GetPodNetworkInfo() (map[string]podNetworkInfo, error)
 			continue
 		}
 
+		ofport, err := oc.ovs.GetOFPort(row["name"])
```

Comment:
you don't need to do this; you should be able to just add `"ofport`" to the list of fields passed to `oc.ovs.Find` above